### ISSUE TITLE
fontLigatures is always true for Fira Code font.

### ISSUE THREAD

**Issue by kumar-anand** [Microsoft/vscode#8253]
- VSCode Version: 1.2.1
- OS Version: Windows 7 Enterprise

Steps to Reproduce:

1. Set the editor font to Fira Code
2. Now try to set the fontLigatures to false (which is default value) but in the editor, it is still working as it is set to true.

Following is the screen grab.

![image](https://cloud.githubusercontent.com/assets/5020381/16409752/bca255c6-3d3c-11e6-8bdf-0e72bf34092f.png)

Regards,
Anand

**Comment by alexandrudima** [comment]
Yes, that is true. I'm a little bit unsure what to do, maybe you have ideas?

* when we added `editor.fontLigatures`, we used a Chromium version that would not do ligatures by default - it would need some special CSS rules to enable ligatures
* in the meantime, we updated to a newer Chromium version that enables ligatures by default
* from an implementation point of view, this makes `editor.fontLigatures` not do anything observable.

I am unsure if we should now write special CSS rules to disable ligatures again by default and enable them only when the setting is set to true.
